code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -112,6 +112,14 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_icon_state = "greenbeerglass"
 	glass_name = "glass of green beer"
 	glass_desc = "A freezing pint of green beer. Festive."
+	
+/datum/reagent/consumable/ethanol/beer/maltliquor
+	name = "Malt Liquor"
+	description = "An alcoholic beverage brewed since ancient times on Old Earth. This variety is super strength, super cheap, and super terrible."
+	boozepwr = 35
```

Comment:
for a "super strong" drink, this is on the lower end of the drink pool when it comes to boozepwr

Reply:
obviously, it's not anywhere near as strong when compared to the other alcoholic beverages, but it's strong when compared to the piss water

Reply:
its 10 boozepwr stronger than beer, this aint nothing special
also you have put this liquid between green beer define and green beer procs for some reason, please move it

Reply:
i changed "super strong" to "stronger than usual" and i moved it to between the light beer and the green beer